pkg/reconciler/taskrun/resources/taskref.go

```diff
@@ -197,6 +198,18 @@ func readRuntimeObjectAsTask(ctx context.Context, obj runtime.Object) (v1beta1.T
 	if task, ok := obj.(v1beta1.TaskObject); ok {
 		return task, nil
 	}
+	if task, ok := obj.(*v1.Task); ok {
```

Comment:
nit: wondering if a switch statement would be preferred here?

Reply:
Thanks!